README.md

```diff
@@ -1,31 +1,39 @@
-# Go stats collection libraries
+# OpenCensus Libraries for Go
 
 [![Build Status][travis-image]][travis-url] [![GoDoc][godoc-image]][godoc-url]
 
-This is still at a very early stage of development and a lot of the API calls
-are in the process of being changed and might break your code in hte future.
+Opencensus-go is a Go implementation of OpenCensus, a toolkit for
+collecting application performance and behavior monitoring data.
+Currently it consists of three major APIs: tags, stats, and tracing.
```

Comment:
it doesn't have tracing yet.

Reply:
Similar to the directories: stats/ and tags/, soon we'll have a trace/
I think it will be good to add a section at the bottom for "## Trace API" with a TODO.

Reply:
Added TODO at the end.